bin/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Usage: bin/run_tests.sh
+#
+# Runs tests.
+#
+# This should be called from inside a container and after the dependent
+# services have been launched. It depends on:
```

Comment:
Does this comment apply anymore if we're running this outside of a container per the "Run tests" CI job in `.github/workflows/build.yaml`?

I see `make test` still runs it from a shell in a Docker container. And the environment variables are using docker aliases for the host names... Now I'm wondering how the above CI is passing still in that case. I guess we set different env vars in that context... Ah I see that we do: https://github.com/mozilla-services/obs-common/blob/4a78aadca50bdf890ebd85a8b0e7993d313a7f08/.github/workflows/build.yaml#L40-L43